interface/Met.hh

```diff
@@ -37,6 +38,10 @@ class Met : public Candidate {
   inline double yy_sig() const { return yy_sig_; }
   /**@}*/
 
+/// The vector of shifted mets
+  inline std::vector<std::pair<std::string,Candidate> > shiftedmets() const { return shiftedmets_; }
```

Comment:
I think you might need to put `ic::Candidate` here, otherwise it's having trouble resolving the name in the dictionary